ext/pdo_mysql/tests/gh15093.phpt

```diff
@@ -0,0 +1,170 @@
+--TEST--
+GH-15093: Add support for all flags from mysql in the PDO MySql driver in the getColumnMeta function.
+--EXTENSIONS--
+pdo_mysql
+--SKIPIF--
+<?php
+require_once __DIR__ . '/inc/mysql_pdo_test.inc';
+MySQLPDOTest::skip();
```

Comment:
```
MySQLPDOTest::skipNotMySQLnd();

if (MYSQLPDOTest::isPDOMySQLnd()) die("skip libmysql only test");
```

By using the above codes, you can separate the tests for mysqlnd and libmysql.
IMHO, these features are less common and less likely to be bug-reported, so it is desirable to be able to cover all cases with tests.

Reply:
I changed the code to use skipNotMySQLnd(), but in retrospect I think you were suggesting to write two separate tests. Given the differences regarding the NUM_FLAG above, separating them does seem like a good idea at first. However, one would expect the two to be functionally equivalent in this test.